src/components/apps/Notes/Notes.svelte

```diff
@@ -0,0 +1,145 @@
+<script>
+  import { theme } from '🍎/stores/theme.store';
+	import {addNote, notesStore} from './stores';
+	
+	const NEW_NOTE = {title: '', text: ''};
+	
+	let editing = false;
+	let note;
+	let selectedId;
```

Comment:
Please add type to this and the other props

Reply:
the types and props are in the ```stores.ts``` file.